x-pack/plugins/infra/public/pages/logs/log_entry_rate/page_results_content.tsx

```diff
@@ -131,12 +127,23 @@ export const LogEntryRateResultsContent: React.FunctionComponent = () => {
     [setAutoRefresh]
   );
 
-  const isFirstUse = useMemo(() => setupStatus === 'hiddenAfterSuccess', [setupStatus]);
+  const hasResults = useMemo(() => (logEntryRate?.histogramBuckets?.length ?? 0) > 0, [
+    logEntryRate,
+  ]);
+
+  const isFirstUse = useMemo(
+    () => setupStatus.type === 'skipped' && !!setupStatus.newlyCreated && !hasResults,
```

Comment:
`setupStatus.type === 'skipped'` has to be checked because it's the tag of the union type. `!hasResults` will be false as soon as the ML job has persisted the first results that match the time range filter.